README.md.tt

```diff
@@ -71,7 +71,7 @@ it first to leverage the `cache_from` settings of Docker Compose which avoids re
 - Build the Docker image:
```

Comment:
Should we also update the instructions above to run the Rails app?

Now it says:

```
- Run the Rails app
```sh
foreman start -f Procfile.dev
```
```

While we can now recommend using `bin/dev`, right?

Reply:
Updated on 046a4f5 ✨